tox.ini

```diff
@@ -12,24 +12,32 @@ deps =
     edx-opaque-keys
     mock
     django_nose>=1.4
+    xblockutils
```

Comment:
I don't think I knew about this back then;
we should be able to pull this directly from `requirements.txt` [1].

[1] https://testrun.org/tox/latest/example/basic.html#depending-on-requirements-txt